test/common/upstream/load_balancer_fuzz_test_utils.h

```diff
@@ -0,0 +1,48 @@
+#include "envoy/config/cluster/v3/cluster.pb.h"
+
+#include "common/upstream/load_balancer_impl.h"
+
+#include "test/mocks/common.h"
+#include "test/mocks/runtime/mocks.h"
+#include "test/mocks/upstream/cluster_info.h"
+#include "test/mocks/upstream/host_set.h"
+#include "test/mocks/upstream/load_balancer_context.h"
+#include "test/mocks/upstream/priority_set.h"
+
+namespace Envoy {
+
+namespace Random {
+class FakeRandomGenerator : public RandomGenerator {
+public:
+  FakeRandomGenerator() = default;
+  ~FakeRandomGenerator() override = default;
+
+  // RandomGenerator
+  uint64_t random() override; // returns a part of the generated bytestring
+  std::string uuid() override { return ""; }
+  uint32_t counter = 0; // Will get modded against byte string length for byte string index
+  std::vector<uint64_t> bytestring_; // String of bytes that returns on random() calls
+};
+} // namespace Random
+
+namespace Upstream {
+namespace {
+
+class LoadBalancerFuzzTestBase {
```

Comment:
what exactly is this used for? is `common_config_` ever used?
are these collections of objects and mocks needed for most load balancers?

can this be a member of the LoadBalancerFuzzBase? 

One way to speed up the fuzzer if this were a member is to make the member static and clear any state per iteration (i don't think runtime_ ever changes in this fuzzer, so why not make the member static?)

Reply:
Common config is explicitly modified in certain unit tests, ex: https://github.com/envoyproxy/envoy/blob/master/test/common/upstream/load_balancer_impl_test.cc#L1086. Thus, it is important to have. I can move this logic to loadbalancerfuzzbase and make it one class. Also, yup static members seem like a good plan. Will take a bullet point list of notes, because a lot of these speed ups map to health_check_fuzz as well! :)

Reply:
Deleted the whole utils class, and moved all the state into my LoadBalancerFuzzBase class.